code/datums/weather/weather_types/snow_storm.dm

```diff
@@ -58,10 +74,52 @@
 
 ///A storm that doesn't stop storming, and is a bit stronger
 /datum/weather/snow_storm/forever_storm
-	telegraph_duration = 0
+	telegraph_duration = 0 SECONDS
 	perpetual = TRUE
 
 	probability = 0
 
 	cooling_lower = 5
 	cooling_upper = 18
+
+/datum/looping_sound/snowstorm
```

Comment:
Why not put this with the rest of the weather looping sounds in `code/datums/looping_sounds/weather.dm`